.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
is the schema particular to vscode IDE?

Reply:
the schema is not particular to vscode. that's why it's been added to `npm run lint`.
it's just that vscode allows the repo to specify a configuration to be loaded by the editor, therefore added in `.vscode`